projects/step-core/src/lib/modules/table/shared/basic-filter-condition.ts

```diff
@@ -0,0 +1,15 @@
+import { FilterCondition } from './filter-condition';
+import { TableRequestFilter } from '../../../client/step-client-module';
+
+export class BasicFilterCondition extends FilterCondition {
+  constructor(private filters: TableRequestFilter[]) {
+    super();
+  }
+  override toRequestFilter(field: string): Array<TableRequestFilter | undefined> {
+    return this.filters;
+  }
+
+  override isEmpty(): boolean {
+    return !this.filters || this.filters.length === 0;
```

Comment:
is there a chance for ```this.filters``` to not exist? I mean the way it is declared, it should always be defined

Reply:
Fixed